### PR TITLE
Support migrate from an existing server

### PR DESCRIPTION
**Pull request by Ericwww**
## Context 

To solve #96 

I think `/palworld/Pal/Saved/Config/LinuxServer/GameUserSettings.ini` is generated after the service is started for the first time, so I can only write an additional script to facilitate migration.

After several experiments, the migration plan was determined to be successful.

## Test instructions

This script is also tested.

image

image

image

## Checklist before requesting a review

* [x] I have performed a self-review of my code
* [x] I've added documentation about this change to the README.
* [x] I've not introduced breaking changes.

### CODE REVIEW

File: README.md
Comment (thijsvanloef) [inline README.md:1]: I prefer to have this in a seperate readme in the migration folder

**Review by thijsvanloef — CHANGES_REQUESTED** [review]

File: README.md
Comment (Ericwww) [reply README.md:1]: OK, separated it.

**Review by thijsvanloef — CHANGES_REQUESTED** [review]
Could you also possibly document the way to do it manually in the readme?

**Comment by RoryKirsi** [comment]
Is any option to work with the Windows server migrate to Linux server? (lock steam_appid.txt)

**Pull request merged**